mediator/med_phases_prep_ocn_mod.F90

```diff
@@ -383,6 +382,13 @@ subroutine med_phases_prep_ocn_custom_cesm(gcomp, rc)
     call ESMF_GridCompGetInternalState(gcomp, is_local, rc)
     if (ChkErr(rc,__LINE__,u_FILE_u)) return
 
+    ! Check that the necessary export field is present
+    if ( .not. FB_fldchk(is_local%wrap%FBExp(compocn), 'Foxx_swnet', rc=rc)) then
+       return
+    end if
+
+    call t_startf('MED:'//subname)
```

Comment:
Shouldn't the t_startf be moved back to the top of the routine?

Reply:
You need this where it is now - in case you need to return. Otherwise the model crashes if you try to return and there is no corresponding call to t_stopf.